src/Commands/Core/eval.js

```diff
@@ -0,0 +1,62 @@
+module.exports = {
+    name: 'eval',
+    aliases: ["ev"],
+    category: 'Developer',
+    description: 'Evaluate javascript code through a command',
+    usage: '<code>',
+    permissions: 'SEND_MESSAGES',
+    clientPerms: 'SEND_MESSAGES',
+    creatorOnly: true,
+    guildOnly: true,
+    premiumOnly: false,
+    requiresArgs: true,
+    run: async (client, msg, args) => {
+        let bot = client;
+
+        let codein = args.join(" ")
+        if(!args[0]) codein = "No input given"
+        msg.channel.send(`**Input:**\n\`\`\`js\n${codein}\n\`\`\``).then(async message => {
+        
+        let authorised = client.creators.ids
```

Comment:
Already a creator permission thing

Reply:
huh?

Reply:
ik, just having that there so deepscan is happy

Reply:
`let authorised = client.creators.ids` doesn't the exist already with line 9.

Reply:
> 
> 
> ik, just having that there so deepscan is happy

confused?